src/content/docs/index.mdx

```diff
@@ -35,3 +35,27 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 		Checkout and contribute to our source code on GitHub
 	</Card>
 </CardGrid>
+
+## Simple as one line
+
+<CardGrid stagger>
+  <Card title="🤖 Using NPM">
+    Boilerplate your app with EruptionJS directly from NPM it's easy as run
+    `npm init eruption@latest`
+  </Card>
+
+  <Card title="🐋 Live from Docker">
+    Or you can try live directly from Docker Compose using
+    `docker-compose -f docker-compose-dev.yml up`
```

Comment:
I'm not quite sure about using the backtick format or the `pre` format here. The first renders inline, the second renders a row that fits 100% of his parent width. 

It seems to me all a design matter in the end, but I think that for sake of legibility, we should go through the `pre` path.